### DISCUSSION TITLE
Custom query - IN operator with binding

### DISCUSSION THREAD

**Discussion by Krolock87** [Budibase/budibase#8326] [Help] [answered]
Hi,

I'm working on a custom query using the IN operator to select my data. It works fine when I enter the required values directly in the query:
![image](https://user-images.githubusercontent.com/116171077/196685209-223552b3-915d-46be-9f43-78dd8c7f29f2.png)

However, using the same values via a binding gives me an empty result:
![image](https://user-images.githubusercontent.com/116171077/196685591-8cf7face-ba46-4c9d-abac-fc46ba444727.png)

Any idea where I am going wrong with this?

**Comment by andz-bb** [comment, marked as answer]
hey @Krolock87, can you try using `true, false` as the binding and removing the parentheses around the binding in the query? In my testing I have found that Budibase is escaping your quotes so the SQL in your example would be hitting the DB as `g.completed in ( '\'true\', \'false\'' )`. 

When you just use `true, false` for your binding it automatically adds quotes and brackets to return `g.completed in ('true','false')`. Hope this helps!

**Reply by Krolock87** [reply]
Hi @andz-bb,

Great, this works! Also tried with different values in the Data Provider configuration and I'm getting the results I expect.
Thanks fpr the help!
